e2e/content-services/pages/permissions.page.ts

```diff
@@ -15,29 +15,97 @@
  * limitations under the License.
  */
 
-import { DataTableComponentPage, AddPermissionsDialogPage, TestElement } from '@alfresco/adf-testing';
+import {
+    AddPermissionsDialogPage,
+    BrowserActions,
+    DataTableComponentPage,
+    DropdownPage,
+    Logger,
+    TestElement
+} from '@alfresco/adf-testing';
 import { browser, by, element } from 'protractor';
 
 export class PermissionsPage {
 
     dataTableComponentPage = new DataTableComponentPage();
     addPermissionsDialog = new AddPermissionsDialogPage();
 
+    rootElement = TestElement.byId('adf-permission-manager-card');
+    emptyElement = new TestElement(this.rootElement.elementFinder.element(by.css('.adf-no-permission__template')));
```

Comment:
this looks weird. Why is this element so complicated?

is something like this not working?
```
emptyElement = TestElement.byCss('.adf-no-permission__template');
```
or
```
rootSelector = 'adf-permission-manager-card';
emptyElement = TestElement.byCss(`${this.rootSelector} .adf-no-permission__template'`);
```